packages/pn-commons/src/components/Data/ItemsTable.tsx

```diff
@@ -1,51 +1,20 @@
-import {
-  Box,
-  Table,
-  TableBody,
-  TableCell,
-  TableContainer,
-  TableHead,
-  TableRow,
-  TableSortLabel,
-} from '@mui/material';
+import React from 'react';
+
+import { Table, TableContainer } from '@mui/material';
 import { styled } from '@mui/material/styles';
-import { visuallyHidden } from '@mui/utils';
-import { ButtonNaked } from '@pagopa/mui-italia';
 
 import { getLocalizedOrDefaultLabel } from '../../services/localization.service';
-import { Column, Item, Sort } from '../../types';
-import { buttonNakedInheritStyle } from '../../utility';
+import ItemsTableBody from './ItemsTable/ItemsTableBody';
+import ItemsTableHeader from './ItemsTable/ItemsTableHeader';
 
-type Props<ColumnId> = {
-  /** Table columns */
-  columns: Array<Column<ColumnId>>;
-  /** Table rows */
-  rows: Array<Item>;
-  /** Table sort */
-  sort?: Sort<ColumnId>;
-  /** The function to be invoked if the user change sorting */
-  onChangeSorting?: (s: Sort<ColumnId>) => void;
+type Props = {
```

Comment:
Try to force the children type.

Check this link https://stackoverflow.com/questions/44475309/how-do-i-restrict-the-type-of-react-children-in-typescript-using-the-newly-adde